docu/files/ModuleExtensibility.html.md

```diff
@@ -8,14 +8,13 @@ order: 10
 
 After you have implemented, deployed, and published your Builder UI module, it is visible in the Builder for tenants who subscribe to the package containing the module. In other words, the basic functionality of the Builder can be extended with additional modules when tenants subscribe to the related packages.
 
-Now, imagine that subscriptions could extend specific Builder UI modules themselves. Imagine that, when a user subscribes to "package A,"  the Builder "module A" is enabled. Additionally, the subscription triggers the declaration of an extension point in "slot A" of module A so that, when the user subscribes to package B, the extension point enriches module A by including a plug-in for module B in slot A.
+The concept of extensibility applies not only to Builder, but also to Builder modules themselves. It's possible to extend Builder modules by means of plug-ins. For example, imagine a user subscribes to "package A". By doing this, the Builder "module A" is subsequently enabled. Module A contains an extension point called "slot A". The subscription to module A also triggers the declaration of an extension point in "slot A". Now, when the user subscribes to a "package B" that has a plug-in Builder module defined for display in module A, slot A, the extension point of slot A enriches module A with a plug-in from module B.
 
 <img src="img/extensibility.png" style="width:600px" class="img-click-modal" alt="Builder Module Extensibility"/>
 
 ### Create a host module
 
-To embed plug-in modules, you should start by defining the host module.
-The host module is basically a regular Builder module, but it should be equipped with an extra JavaScript API that supports the Builder plug-in mechanism. It should also define slots for the plug-ins.
+To embed plug-in modules, start by defining the host module. The host module is basically a regular Builder module. However, it should have an extra JavaScript API that supports the Builder plug-in mechanism. It should also define slots for the plug-ins.
```

Comment:
Per the standards, please avoid "should". It can create confusion about who provides what "should" be there. You can use command tense in whatever way makes sense. For example, "The host module is basically a regular Builder module. However, when you create a host module for embedding plug-in modules, create it with an extra JavaScript API that supports the Builder plug-in mechanism and defines the slots for the plug-ins."